### PR TITLE
fix: Avoids throwing an error with subscription-less zone-cps

### PR DESCRIPTION
**Pull request by johncowen**
I noticed in some logs that we were throwing an error here depending on whether subscriptions are defined or not.

I fixed the bug and added a more user facing graceful warning in the application code, but also made a few updates to some tests related files while I was there.

Signed-off-by: John Cowen <john.cowen@konghq.com>

### CODE REVIEW

File: features/zones/DetailViewContent.feature [hunk 1]
```diff
@@ -4,38 +4,6 @@ Feature: Zones: Detail view content
...
-    Then the "$details" element contains "ZoneOverview"
-    Then the "$details" element contains "online"
-    Then the "$details" element contains "dpToken"
-
-    When I click the "#insights-tab" element
-    Then the "$details" element contains "Connect time: July 28, 2020 at 4:18:09 PM"
-
-    When I click the "#config-tab" element
-    Then the "$details" element contains "adminAccessLogPath"
-
```
Comment (johncowen) [inline features/zones/DetailViewContent.feature:38]: I moved this into `features/zones/zone-cps/Item.feature` while I was here (we should also move the others to `features/zones/zone-ingresses` and `features/zones/zone-egresses` at some point)

File: src/app/zones/components/ZoneDetails.vue [hunk 2]
```diff
@@ -168,10 +177,12 @@ const warnings = computed<ZoneCompatibility[]>(() => {
 
 const codeOutput = computed(() => {
   const subscriptions = props.zoneOverview.zoneInsight?.subscriptions ?? []
-  const lastSubscription = subscriptions[subscriptions.length - 1]
+  if (subscriptions.length > 0) {
```
Comment (johncowen) [inline src/app/zones/components/ZoneDetails.vue:180]: This conditional/guard is the bug fix

File: src/app/zones/locales/en-us/index.yaml [hunk 3]
```diff
@@ -3,6 +3,8 @@ zone-cps:
     item:
       title: "{name} Zone CP"
       breadcrumbs: Zone CPs
+      config:
+        no-subscriptions: This zone has no subscriptions
```
Comment (johncowen) [inline src/app/zones/locales/en-us/index.yaml:7]: If someone knows some better text we can amend that here or later.

**Review by johncowen — COMMENTED** [review]
Adding inline notes

**Review by lahabana — APPROVED** [review]

**Pull request merged**